forge-gui/res/cardsfolder/n/noble_heritage.txt

```diff
@@ -9,6 +9,6 @@ SVar:DBChoice:DB$ ChooseCard | Defined$ Player.IsRemembered | Amount$ 1 | MinAmo
 SVar:DBPutCounter:DB$ PutCounterAll | ValidCards$ Creature.IsRemembered | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBRememberPlayer
 SVar:DBRememberPlayer:DB$ Pump | RememberObjects$ RememberedController | SubAbility$ DBRepeatEach
 SVar:DBRepeatEach:DB$ RepeatEach | RepeatPlayers$ Opponent.IsRemembered | RepeatSubAbility$ DBPump | SubAbility$ DBCleanup
```

Comment:
Imo `RepeatPlayers$ RememberedController` should work so you can at least get rid of the Pump above there

Nice work on getting rid of TiedToKeyword, I was suspecting it was no longer needed

Reply:
For this, PumpEffect should probably be extended to turn one Keyword into multiple extra ones like StaticAbility can